lib/tracer.js

```diff
@@ -3,6 +3,8 @@
 // We don't want to require strong-agent, because anything it requires will
 // become untraceable. Still, we need to use its idea of the appName, and access
 // lrtime from the addon, so require the agent internals.
+return ;
```

Comment:
add a lint configuration comment: http://eslint.org/docs/user-guide/configuring#configuring-rules

also, put the return as the first line in the file